demisto_sdk/commands/common/errors.py

```diff
@@ -115,6 +115,7 @@
     "playbook_cant_have_deletecontext_all": "PB105",
     "using_instance_in_playbook": "PB106",
     "invalid_script_id": "PB107",
+    "invalid_script_using_commands": "SC100",
```

Comment:
we already have SC100 and 101:
![image](https://user-images.githubusercontent.com/50295826/103742061-e3fb8380-5002-11eb-8f14-ecb7b3744c17.png)